MagicSpell/InstrPageViewController.swift

```diff
@@ -54,22 +54,18 @@ class InstrPageViewController: UIViewController{
         labelForSubtitle.autoresizingMask = [.FlexibleLeftMargin, .FlexibleRightMargin]
         self.view.addSubview(labelForSubtitle)
         
-        imageViewForLeft = UIImageView()
-        imageViewForLeft.image = UIImage(named: "instLeft")
-        imageViewForLeft.setWidth(230)
-        imageViewForLeft.setHeight(200)
-        imageViewForLeft.setRight(UIScreen.mainScreen().bounds.size.width/2 - 40)
-        imageViewForLeft.setTop(labelForSubtitle.bottom + 100)
-        self.view.addSubview(imageViewForLeft)
-        
-        imageViewForRight = UIImageView()
-        imageViewForRight.image = UIImage(named: "instRight")
-        imageViewForRight.setWidth(270)
-        imageViewForRight.setHeight(120)
-        imageViewForRight.setLeft(UIScreen.mainScreen().bounds.size.width/2 + 40)
-        imageViewForRight.setTop(imageViewForLeft.height/2+imageViewForLeft.top - imageViewForRight.height/2)
-        self.view.addSubview(imageViewForRight)
-        
+        imageAnimation = UIImageView()
```

Comment:
Consider using Auto Layout, doesn't have to be done in this pull request. I see a lot of magic numbers here: 400, 400, 80)